CrimeMonitor/Properties/CrimeMonitor.resx

```diff
@@ -165,6 +165,9 @@
   <data name="orbital_station_checkbox" xml:space="preserve">
     <value>Prioritize Orbital Stations:</value>
   </data>
+  <data name="p1" xml:space="preserve">
+    <value>Note: Restrictive station distance setting may significanlty increase search time</value>
```

Comment:
Typo: should be `significantly`.

Reply:
Proposed revision: `Tip: Maximize the maximum station distance to minimize search times.`

Reply:
Agreed. Done.